vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -20,7 +20,11 @@
 
 namespace ledger {
 
-extern bool is_production;
+using VisitData = ledger::mojom::VisitData;
```

Comment:
`ledger::` not needed as already in `ledger` namespace

Reply:
this was not rebased correctly as all mojo stuff in now in `mojo_structs` file

Reply:
Fixed